content/docs/about/productionusage.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Production Usage"
+weight: 14
+description: >
+  kube-vip usage
+---
+
+
+## kube-vip Maturity
+
+The kube-vip project is deployed and used in a number of production environments and is included/bundled with a number of production Kubernetes based platforms. Whilst kube-vip could be thought of as pre-release, it has happily found it's home in places where it has ran production workloads. I cursary glance at time of writing this (mid-July 2022) also reveals that downloads combined from Docker Hub and GitHub container registry amount to around 19,000,000 kube-vip container pulls. 
```

Comment:
First sentence: "...it has happily found its home in places where it runs production workloads."
Second sentence: "A cursory glance at this time of writing (July 2022) also..."